### PR TITLE
feat : add home page

### PR DESCRIPTION
**Pull request by Kevin-Solomon**
# Home page of the eCommerce App
image
[CodeSandbox link for Home page](https://9pluj.csb.app/pages/home/home.html)

### CODE REVIEW

**Review by starver20 — APPROVED** [review]
Looks good to go.👍🏻

File: pages/home/home.html [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+      <div class="grid-aside">
+        <img class='responsive-img' src="https://images.unsplash.com/photo-1643133277936-9f93d8792522?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHx0b3BpYy1mZWVkfDMyfHhqUFI0aGxrQkdBfHxlbnwwfHx8fA%3D%3D&auto=format&fit=crop&w=500&q=60">
+      </div>
+      <div class="grid-aside">
+        <img class='responsive-img' src="https://images.unsplash.com/photo-1643133277936-9f93d8792522?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHx0b3BpYy1mZWVkfDMyfHhqUFI0aGxrQkdBfHxlbnwwfHx8fA%3D%3D&auto=format&fit=crop&w=500&q=60">
+      </div>
+      <div class="grid-aside">
+        <img class='responsive-img' src="https://images.unsplash.com/photo-1643133277936-9f93d8792522?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHx0b3BpYy1mZWVkfDMyfHhqUFI0aGxrQkdBfHxlbnwwfHx8fA%3D%3D&auto=format&fit=crop&w=500&q=60">
+      </div>
+    </div>
```
Comment (ajitpsakri) [inline pages/home/home.html]: On screens, they have given text on the image. I guess that's a requirement.
image

**Review by sagar455 — APPROVED** [review]
looks good.

File: pages/home/home.html [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    <main class='main-content'>
+      <img class="responsive-img" src="https://images.unsplash.com/photo-1607083206869-4c7672e72a8a?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8Nnx8c2FsZXxlbnwwfHwwfHw%3D&auto=format&fit=crop&w=500&q=60"> 
+    </main>    
+    
+<!-- add grid and grid-column-2 to the class of parent -->
+
+    <div class="grid grid-column-2">
```
Comment (ajitpsakri) [inline pages/home/home.html:97]: `gap` between `grid-aside` and `grid-main-content` can be zero, that would look good

File: pages/home/home.css [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+.main-content .responsive-img {
+  block-size: inherit;
+}
+.card {
+  width: 100%;
+}
+.navbar .navbar-btn {
+  display: none;
+}
+@media (max-width: 768px) {
```
Comment (ajitpsakri) [inline pages/home/home.css:42]: Like how your thinking of making it responsive from beginning

**Review by ajitpsakri — APPROVED** [review]

File: pages/home/home.html (hunk 2 above)
Comment (Kevin-Solomon) [reply pages/home/home.html:97]: In the design file there was a gap between the two elements

File: pages/home/home.html (hunk 1 above)
Comment (Kevin-Solomon) [reply pages/home/home.html]: > On screens, they have given text on the image. I guess that's a requirement. image

Done

**Pull request merged**